src/shared/utils/fetchApi.js

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+
+async function fetchApi({ url, method, body, propertyName }) {
```

Comment:
I liked the implementation of this util function.

Reply:
thank you